FAQ.md

```diff
@@ -10,26 +10,43 @@ Feel free to add new questions or send comments to this page as pull requests._
 >- Or a single/few static IPs?
 >- Will one node be the designated WAN egress/ingress point or will there be multiple gateways on the meshnet?
 
+@jonah-archive is working with the venue and local ISPs to pool together some internets. I am imagining the network to primarily operate without a WAN connection, but expect one or more more nodes as gateway depending on how the connectivity is sourced. If there is a line that pops out somewhere vs. two providers where one is satellite, we could have multiple gateways.
+
+I don't expect static IPs would be needed as I don't expect to be hosting services to the outside world. Of course, if people run some distributed service with Internet peers, that is still possible.
+
 >Since the location is coastal (e.g. wind, moisture, dense fog, animals, etc.), what are the allowances for HA - high availability - radios, routers/switches
 >- You can use simple STP for fail-over between radios
 >- 900Mhz can be a useful--if quite slow--backup
+>- Be wary of using any 60Ghz PtP option as fog can really hinder link performance
+
+We need the core areas to have redundancy against at least one radio failure, and have ways to monitor the overall link status of the network.
+
+We should get some [disaster.radio](https://disaster.radio) distributed at key locations, perhaps as a separate network where people can connect to a camp-global chat.
 
->Be wary of using any 60Ghz PtP option as fog can really hinder link performance
+Currently not using any 60 GHz equipment, but even if we do they'd be used (~ 250 m) at much smaller distances than specifications (1-2 km).
 
 >Is there power at each desired "node" location?
 >- If not, what is planned?
 >- POE beyond 100-150 feet is unreliable
 
+Not all locations have power. I believe the area with buildings and around the lake has power sources. At the end of March we will visit the site to identify power requirements. We also expect power build-out to complete before meshnet build-out.
+
 >Services
 >- Are any mesh services expecting to be reachable from the public internet?
 >- Waterproof/moisture-proof cases for Pi-like devices is crucial
```

Comment:
I am most concerned about the outdoor readiness of the ClearFog Pro, as I am hoping to use that as the main router. I am waiting for responses from their office about a couple things.

Reply:
Are you concerned about it being weather resistant? Grabbing a simple NEMA-rated box and plumbing cabling with water ingress in mind should do. 

[Something like this maybe](https://www.amazon.com/Altelix-Fiberglass-Weatherproof-Enclosure-Stainless/dp/B079HG77S6/ref=sr_1_3?keywords=outdoor+network+enclosure+box&qid=1552755356&s=gateway&sr=8-3).

Also, our more remote wireless sites sometimes get ahold of old, used traffic signaling cabinets from CalTrans or city departments. But, that may be a little too large....!

Reply:
Added to https://github.com/dweb-camp-2019/meshnet/issues/8